src/components/AssignmentFeedbackForm/AssignmentFeedbackForm.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  InputWrapper,
+  Slider,
+  Space,
+  TextInput,
+  Checkbox,
+  Text,
+} from '@mantine/core';
+import { useForm } from '@mantine/hooks';
+import useAssignment from '../../context/AssignmentContext';
+import GenericButton from '../generics/GenericButton';
+
+const easyMarks = [
+  { value: 0, label: 'Extremely Difficult' },
+  { value: 25, label: 'Difficult' },
+  { value: 50, label: 'Neutral' },
+  { value: 75, label: 'Easy' },
+  { value: 100, label: 'Extremely Easy' },
+];
+
+const usefulMarks = [
+  { value: 0, label: 'Extremely Useless' },
+  { value: 25, label: 'Useless' },
+  { value: 50, label: 'Neutral' },
+  { value: 75, label: 'Useful' },
+  { value: 100, label: 'Extremely Useful' },
+];
+
+interface IFormValues {
+  easiness: number;
+  usefulness: number;
+  whatWorkedWell: string;
+  whatWentWrong: string;
+  useAgain: boolean;
+}
+
+const initialValues: IFormValues = {
+  easiness: 0,
+  usefulness: 0,
+  whatWorkedWell: '',
+  whatWentWrong: '',
+  useAgain: false,
+};
+
+const AssignmentFeedbackForm = () => {
+  const { updateUserAssignmentDoc } = useAssignment();
+
+  const form = useForm({
+    initialValues,
+  });
+
+  const onSubmit = (values: IFormValues) => {
+    updateUserAssignmentDoc!({ submitted: true, feedback: values });
+  };
+
+  return (
+    <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+      <Text>
+        Please complete this short questionaire. Once you&apos;re done we will
+        let you download your answers so you can submit to coderunner.
+        <br />
+        Important: you must submit to coderunner to get your marks for this
+        course!
+      </Text>
+      <InputWrapper label='How easy was it to use this tool?' required>
+        <Slider
+          label={(val) => easyMarks?.find((mark) => mark.value === val)?.label}
+          step={5}
+          marks={easyMarks}
+          // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
Could you just disable this in the eslint config file rules?
`'react/jsx-props-no-spreading': 'off'`